src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -162,6 +189,26 @@ public void verifyInstructorEditFormDisabled(int instrNum) {
         assertTrue(isNotEditable);
     }
 
+    /*
+     * newDisplayName is verified depending on newIsDisplayedToStudents.
```

Comment:
You can use `{@code newDisplayName}` and `{@code newIsDisplayedToStudents}` to refer to the parameters.